src/translations/en.js

```diff
@@ -17,6 +17,8 @@ module.exports = {
   fieldInvalidForPostalCode: 'This postal code is not valid.',
   genericError: 'Something went wrong on our end.',
   hostedFieldsFailedTokenizationError: 'Please check your information and try again.',
+  hostedFieldsTokenizationFailOnDuplicateError: 'This credit card is already saved to a customer.',
```

Comment:
Wasn't sure how to word this. 

When the the failOnDuplicatePaymentMethod option is passed in client token generation, it will fail if the card already exists in the merchant's vault, even if the customer is different.